src/main/java/core/basesyntax/ShopService.java

```diff
@@ -0,0 +1,5 @@
+import java.util.List;
```

Comment:
It seems that the `FruitTransaction` class is not imported in this file. Make sure to import the necessary classes to avoid compilation errors.